examples/step-9/step-9.cc

```diff
@@ -1319,15 +1320,23 @@ namespace Step9
 
 // @sect3{Main function}
 
-// The <code>main</code> function is exactly like in previous examples, with
-// the only difference in the name of the main class that actually does the
-// computation.
+// The <code>main</code> function is similar to the previous examples. The main 
+// difference is that we use <code>MultithreadInfo</code> to set the maximum 
+// number of threads. The number of threads used is the minimum of the 
+// environment variable DEAL_II_NUM_THREADS and the parameter of 
+// <code>set_thread_limit</code>. If no value is given to
+// <code>set_thread_limit</code>, the default value from TBB is used. If 
+// <code>set_thread_limit</code> is omitted, the number of threads will be 
+// chosen by TBB indepently of DEAL_II_NUM_THREADS.
 int main ()
 {
   try
     {
       dealii::deallog.depth_console (0);
 
+      dealii::MultithreadInfo multithread_info;
+      multithread_info.set_thread_limit();
```

Comment:
There is a global variable ```multithread_info``` you can use, so just delete the first line and it should work.